### PR TITLE
Fix checkstyle violations in 11 modules

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
After a short break, again some work on #2238

- `statistics-export-service-impl`
- `statistics-provider-influx`
- `statistics-service-api`
- `statistics-service-impl`
- `statistics-service-remote`
- `termination-state-aws`
- `termination-state-impl`
- `smil-impl`
- `runtime-info`
- `live-schedule-impl`
- `authorization-xacml`

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/#development-process/#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java [hunk 1]
```diff
@@ -116,11 +116,13 @@
   private static final String DEFAULT_LIVE_TARGET_FLAVORS = "presenter/delivery";
   static final String DEFAULT_LIVE_DISTRIBUTION_SERVICE = "download";
 
+  // CHECKSTYLE:OFF
```
Comment (lkiesow) [inline modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java:120]:
```suggestion
  // Deactivating checkstyle to preserve the long URL
  // CHECKSTYLE:OFF
```

**Review by lkiesow — APPROVED** [review]
Looks good.
@KatrinIhler, you want to review this as well?
Or can I merge this?

File: modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java (hunk 1 above)
Comment (LukasKalbertodt) [reply modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java:120]: Fixed (and rebased)

**Comment by KatrinIhler** [comment]
Since Lars approved, I'm gonna merge this before I procrastinate the review anymore and hold everything up ;)

**Pull request merged**